packages/local_storage_todos_api/test/src/local_storage_todos_api_test.dart

```diff
@@ -0,0 +1,170 @@
+// ignore_for_file: prefer_const_constructors
+import 'dart:convert';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:local_storage_todos_api/local_storage_todos_api.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:todos_api/todos_api.dart';
+
+class SharedPreferencesMock extends Mock implements SharedPreferences {}
+
+void main() {
+  late SharedPreferences sharedPreferences;
+  late LocalStorageTodosApi todosApi;
+
+  setUp(() {
+    sharedPreferences = SharedPreferencesMock();
+    todosApi = LocalStorageTodosApi(plugin: sharedPreferences);
+  });
+
+  group('LocalStorageTodosApi', () {
+    test('can be instantiated', () {
+      expect(LocalStorageTodosApi(plugin: sharedPreferences), isNotNull);
```

Comment:
```suggestion
      expect(() => LocalStorageTodosApi(plugin: sharedPreferences), returnsNormally);
```

This is alternative approach that is more explicit about what we test. It will never be null, but we want to make sure it does not throw.